### ISSUE TITLE
use proxy in livereload emulate

### ISSUE THREAD

**Issue by riderx** [mwaylabs/generator-m-ionic#446]
the options --proxy do nothing in this line :
`gulp --proxy --livereload "emulate ios"`

**Comment by gruppjo** [comment]
the proxy is not designed to be run with livereload. Why do you want to use the proxy on the device? When you're running the task on the device, CORS isn't usually an issue.

**Comment by voznik** [comment]
because this `--liverolad` command serves using browser-sync, I also have CORS issues, and I wish `--proxy` to fix this

**Comment by gruppjo** [comment]
@voznik, of course! I'm currently not able to invest time in this because I'm working on the next version of the generator for ionic2, but this seems like a useful and legit feature!
